src/schema/game_state.py

```diff
@@ -79,6 +79,10 @@ class GameState(BaseModel):
         description="The key of the last question asked to the user via context_utils.await_ask.",
     )
 
+    onboarding_agent_has_completed: Optional[bool] = Field(
```

Comment:
be more explicit about whether the onboarding agent has run. before we were sort of duck-typing it (see below)